### DISCUSSION TITLE
[Implemented] Specification for `plotPopulationTimeProfile()`

### DISCUSSION THREAD

**Discussion by PavelBal** [Open-Systems-Pharmacology/OSPSuite-R#832] [Ideas]
Related to #674 

# Signature

```r
# `...` will be parameters to control annotations and aesthetics of the plot
plotPopulationTimeProfile(dataCombined, ...)
```

# Implementation plan

```r
data <- dataCombined$toDataFrame()

tlf::plotTimeProfile(data, ...) 
```

# Rough expected output:

![image](https://user-images.githubusercontent.com/11330453/158802261-bd18f90d-efe7-427b-b86e-71d0c1a16854.png)

# Decision points:

- only `quantiles = c(0.05, 0.50, 0.95)` parameter or some other parameter?

# Tasks

Each task will get its own PR.

- Implement `plotPopulationTimeProfile()`

**Comment by Yuri05** [comment]
@sfrechen @AnnikaRPS @twendl @incei @christoph27 @andredlm FYI
(This is about creation of plots directly in the ospsuite R package, not as part of reports made by RE)

**Comment by PavelBal** [comment]
@IndrajeetPatil Example of population simulation are in the file `Stevens_2012_placebo_pop_results.csv`, you can load them the same way you are loading the individual results `Stevens_2012_placebo_indiv_results.csv`. They were used to produce the expected output figures in this discussion.

**Comment by IndrajeetPatil** [comment]
`plotIndividualTimeProfile` and `plotPopulationTimeProfile` line type should be solid.
